File: 코딩 기초 트레이닝/문자열 뒤집기-p181905/문자열 뒤집기.py
# https://school.programmers.co.kr/learn/courses/30/lessons/181905

def solution(my_string, s, e):
    answer = my_string[0:s]
    tmp = my_string[s:e+1]
    tmp = tmp[::-1]
    answer += tmp
    answer += my_string[e+1:]

    return answer

File: 코딩 기초 트레이닝/문자열 뒤집기-p181905/문자열 뒤집기.js
// https://school.programmers.co.kr/learn/courses/30/lessons/181905

function solution(my_string, s, e) {
    var answer = my_string.slice(0, s);
    let tmp = my_string.slice(s, e+1).split("").reverse().join('');
    answer += tmp;
    answer += my_string.slice(e+1, );
    return answer;
}
